sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -4309,14 +5745,86 @@
             "inputType": "hidden",
             "id": "em1y8gd",
             "defaultValue": "",
-            "dataGridLabel": false
+            "dataGridLabel": false,
+            "addons": []
+        },
+        {
+            "label": "applicationStatus",
+            "customClass": "",
+            "modalEdit": false,
+            "defaultValue": null,
+            "persistent": true,
+            "protected": false,
+            "dbIndex": false,
+            "encrypted": false,
+            "redrawOn": "",
+            "customDefaultValue": "",
+            "calculateValue": "",
+            "calculateServer": false,
+            "key": "applicationStatus",
+            "tags": [],
+            "properties": {},
+            "logic": [],
+            "attributes": {},
+            "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+            },
+            "type": "hidden",
+            "input": true,
+            "tableView": false,
+            "placeholder": "",
+            "prefix": "",
+            "suffix": "",
+            "multiple": false,
+            "unique": false,
+            "hidden": false,
+            "clearOnHide": true,
+            "refreshOn": "",
+            "dataGridLabel": false,
+            "labelPosition": "top",
+            "description": "",
+            "errorLabel": "",
+            "tooltip": "",
+            "hideLabel": false,
+            "tabindex": "",
+            "disabled": false,
+            "autofocus": false,
+            "widget": {
+                "type": "input"
+            },
+            "validateOn": "change",
+            "validate": {
+                "required": false,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+            },
+            "conditional": {
+                "show": null,
+                "when": null,
+                "eq": ""
+            },
+            "allowCalculateOverride": false,
+            "showCharCount": false,
+            "showWordCount": false,
+            "allowMultipleMasks": false,
+            "inputType": "hidden",
+            "id": "e6z1qd9",
+            "addons": []
         }
     ],
     "display": "form",
     "title": "Report Scholastic Standing Change",
     "name": "reportscholasticstandingchange",
     "path": "reportscholasticstandingchange",
     "created": "2021-09-13T18:48:47.454Z",
-    "modified": "2021-10-14T19:44:12.505Z",
+    "modified": "2022-04-12T21:00:30.803Z",
```

Comment:
Can we please mark the below hidden fields as not persisted on the server to avoid them to be saved on the dynamic data on DB.
![image](https://user-images.githubusercontent.com/61259237/163863223-63555ee7-aa3d-4b81-a35d-58c5147c6de1.png)
![image](https://user-images.githubusercontent.com/61259237/163863292-8ed2c5a3-3695-433a-b30e-2f2032b07697.png)